core/themes/basis/css/component/node.css

```diff
@@ -0,0 +1,25 @@
+/**
+ * @file
+ * Styling for node content.
+ */
+
+.node {
+  position: relative;
+}
+
+/* Display "Draft" watermark on unpublished content. */
+/* See basis_preprocess_node(). */
+.node[data-status]:before {
+  content: attr(data-status);
+  position: absolute;
+  z-index: 5;
+  width: 100%;
+  top: 20%;
+  pointer-events: none;
+  text-align: center;
+  text-transform: uppercase;
+  font-size: 10em;
+  font-weight: bold;
+  opacity: 0.5;
+  color: #dee2ea;
```

Comment:
One potential issue, if the user's BG color is the color we set this text to they won't see it. We could play it safer and add a text-shadow of a different color for instance this ought to do it: 
`text-shadow: 0 0 10px #fff, 0 0 10px #fff;`